doc/FAQ.md

```diff
@@ -14,6 +14,8 @@ User
 * **[Can I subscribe to a hashtag?](help/FAQ#hashtag)**
 * **[How to create a RSS feed of the stream?](help/FAQ#rss)**
 * **[Are there any clients for friendica I can use?](help/FAQ#clients)**
+* **[How can I share my color scheme with others?](help/FAQ#schemesharing)**
```

Comment:
I don't think this should be documented in the FAQ. It should rather be documented in the settings if the frio theme is used.

Or you need to implement color schemes for all other themes as well.

Reply:
Thinking about it now, you're right. And I also think the note given below the share scheme box in settings is clear enough.